Разметка главной страницы

OliaErmak opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/315091/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/315091-device-33/2/

OliaErmak committed:
Выделяет крупные смысловые блоки

Добавила блоки header, main, footer

OliaErmak committed:
Добавляет в блоках крупные смысловые разделы

Добавила крупные смысловые разделы в блоках header, main, footer

OliaErmak committed:
Добавляет разметку главной страницы

ad0v0 commented:
http://joxi.ru/GrqaGqvuz6GqWr пожалуйста, обязательно публикуй сборку перед отправкой задания на проверку 🙏🏻

ad0v0 commented:
**Проверка разметки главной и каталога:**

1) Проверка задания согласно критериям:
- ✅ [PROJ-01. Кодгайд](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-01) - ок
- ☑️ [PROJ-02. Именование](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-02):
-- http://joxi.ru/brRxKNQCB3LJq2 bag => cart, enter => login
- ✅ [PROJ-03. Название папок и файлов](https://up.htmlacademy.ru/htmlcss/32/criteries#PROJ-03) - ок 
- ✅ [HTML-01. Ориентиры](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-01) - ок 
- ☑️ [HTML-02. Заголовки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-02):
-- http://joxi.ru/Y2LaQxKuxdEajA полноценная навигация находится в шапке
-- http://joxi.ru/4Ak6l18CjRkZQ2 называем заголовки исходя из наполнения смыслового блока. Например, промо товаров, а дальше название товара
- ✅ [HTML-03. Кнопки и ссылки](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-03) - ок 
- ☑️ [HTML-04. Работоспособные формы](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-04):
-- http://joxi.ru/ZrJKkndCQ6n5Vm заменить на метод get, поскольку пользователю нужно получить ответ от сервака по своему запросу
- ☑️ [HTML-05. Подписи полей форм](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-05):
-- http://joxi.ru/LmGWnR5UBngeMm добавить подпись
- ✅ [HTML-07. Валидация HTML](https://up.htmlacademy.ru/htmlcss/32/criteries#HTML-07) - ок 
- ☑️ [IMG-01. Контентная и декоративная графика](https://up.htmlacademy.ru/htmlcss/32/criteries#IMG-01):
-- http://joxi.ru/zAN361ac1K8j8r логотипы это всегда контентное изображение, поскольку являются айдентикой бренда 
- ☑️ [CONT-01. Текстовые переносы](https://up.htmlacademy.ru/htmlcss/32/criteries#CONT-01):
-- http://joxi.ru/n2YJXEZUkneP5A давай пока что попробуем обойтись без них, а уже на этапе декоративных элементов добавим там, где они действительно нужны и где мы не можем реализовать перенос на новую строку 
- ✅ [ALLY-01. Описание графики](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-01) - ок 
- ✅ [ALLY-02. Подписи интерактивных элементов](https://up.htmlacademy.ru/htmlcss/32/criteries#ALLY-02) - ок 
2) Разное:
- ☑️ http://joxi.ru/a2XNn0KiQVll7r эти элементы отлично подходят для списка, поскольку являются единообразными по своей сути элементами 
3) Общее впечатление от выполненной работы: очень классно чистый и аккуратный код! Молодец 👍🏻 

🎌 - есть замечание, надо править
➡️ - есть комментарий, желательно учесть
✅ - выполнено 
☑️ - правка по замечанию внесена
❓ - есть вопрос, желательно ответить

ПС Ошибки , которые дублируются в коде, в замечаниях не дублирую, но перепроверять и править нужно везде и обязательно

OliaErmak committed:
Исправляет разметку главной страницы

Исправила после ревью наставницы.

Немного исправила списки в коде и их классы, по одинаковой схеме.

OliaErmak committed:
Исправила разметку главной страницы #2

Забыла добавить иконку HTML Academy в футере как контекстное изображение

OliaErmak committed:
Исправила разметку#3

Изменила с section на div контейнер с ссылками в футере

ad0v0 commented:
принимаю!
